transform/llvm.go

```diff
@@ -75,26 +75,14 @@ func replaceGlobalIntWithArray(mod llvm.Module, name string, buf interface{}) ll
 	return global
 }
 
-// typeHasPointers returns whether this type is a pointer or contains pointers.
```

Comment:
This seems like a useful function, even if it's unused at the moment.  Do we have an equivalent function to answer this question?

Reply:
I don't think so. But if needed, we can always bring it back from the git history. I don't like keeping dead code around.